### PR TITLE
doc changes for tiup deploy multiple ticdc

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Doc change for https://internal.pingcap.net/jira/browse/FD-991 
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-cluster-topology-reference.md [hunk 1]
```diff
@@ -391,6 +391,7 @@ drainer_servers:
 - `os`：`host` 字段所指定的机器的操作系统，若不指定该字段，则默认为 `global` 中的 `os`
 - `arch`：`host` 字段所指定的机器的架构，若不指定该字段，则默认为 `global` 中的 `arch`
 - `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`
+- `ticdc_cluster_id`：指定该服务器对应的 TiCDC 集群 ID。若不指定该字段，则自动加入默认 TiCDC 集群，该配置只在 6.3 及以上 TiDB 版本中才生效。
```
Comment (ran-huang) [inline tiup/tiup-cluster-topology-reference.md:394]:
```suggestion
- `ticdc_cluster_id`：指定该服务器对应的 TiCDC 集群 ID。若不指定该字段，则自动加入默认 TiCDC 集群，该配置只在 v6.3.0 及以上 TiDB 版本中才生效。
```

**Review by ran-huang — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md [hunk 2]
```diff
@@ -391,6 +391,7 @@ drainer_servers:
 - `os`：`host` 字段所指定的机器的操作系统，若不指定该字段，则默认为 `global` 中的 `os`
 - `arch`：`host` 字段所指定的机器的架构，若不指定该字段，则默认为 `global` 中的 `arch`
 - `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`
+- `ticdc_cluster_id`：指定该服务器对应的 TiCDC 集群 ID。若不指定该字段，则自动加入默认 TiCDC 集群，该配置只在 v6.3.0 及以上 TiDB 版本中才生效。
```
Comment (Oreoxmt) [inline tiup/tiup-cluster-topology-reference.md:394]:
```suggestion
- `ticdc_cluster_id`：指定该服务器对应的 TiCDC 集群 ID。若不指定该字段，则自动加入默认 TiCDC 集群。该配置只在 v6.3.0 及以上 TiDB 版本中才生效。
```

File: config-templates/complex-cdc.yaml [hunk 3]
```diff
@@ -99,18 +99,21 @@ cdc_servers:
     data_dir: "/tidb-data/cdc-8300"
     log_dir: "/tidb-deploy/cdc-8300/log"
     # gc-ttl: 86400 
+    # ticdc_cluster_id : cluster1
```
Comment (Oreoxmt) [inline config-templates/complex-cdc.yaml:102]:
```suggestion
    # ticdc_cluster_id: cluster1
```

File: config-templates/complex-cdc.yaml [hunk 4]
```diff
@@ -99,18 +99,21 @@ cdc_servers:
...
   - host: 10.0.1.2
     port: 8300
     deploy_dir: "/tidb-deploy/cdc-8300"
     data_dir: "/tidb-data/cdc-8300"
     log_dir: "/tidb-deploy/cdc-8300/log"
     # gc-ttl: 86400 
+    # ticdc_cluster_id : cluster1
```
Comment (Oreoxmt) [inline config-templates/complex-cdc.yaml:109]:
```suggestion
    # ticdc_cluster_id: cluster1
```

File: config-templates/complex-cdc.yaml [hunk 5]
```diff
@@ -99,18 +99,21 @@ cdc_servers:
...
   - host: 10.0.1.3
     port: 8300
     deploy_dir: "/tidb-deploy/cdc-8300"
     data_dir: "/tidb-data/cdc-8300"
     log_dir: "/tidb-deploy/cdc-8300/log"
     # gc-ttl: 86400 
+    # ticdc_cluster_id : cluster2
```
Comment (Oreoxmt) [inline config-templates/complex-cdc.yaml:116]:
```suggestion
    # ticdc_cluster_id: cluster2
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Comment by ran-huang** [comment]
@nexustar @leoppro PTAL

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: config-templates/complex-cdc.yaml [hunk 6]
```diff
@@ -99,18 +99,21 @@ cdc_servers:
     data_dir: "/tidb-data/cdc-8300"
     log_dir: "/tidb-deploy/cdc-8300/log"
     # gc-ttl: 86400 
+    # ticdc_cluster_id: cluster1
```
Comment (nexustar) [inline config-templates/complex-cdc.yaml:102]: should be "cluster1"

**Review by nexustar — APPROVED** [review]

**Pull request merged**